frontend/src/components/StyledLink/StyledLink.js

```diff
@@ -2,6 +2,7 @@ import styled from 'styled-components';
 import { Link } from 'react-router-dom';
 
 const StyledLink = styled(Link)`
+  width: 100%;
```

Comment:
Jeśli inne komponenty z tego korzystają to zmiana tej wartości może popsuć je :)